apps/css-workshop/src/components/Tag.astro

```diff
@@ -0,0 +1,41 @@
+---
+const { title, className, id, style, button = true, variant, href, ...props } = Astro.props;
+---
+
+<>
+  {
+    variant === 'basic' && !href && (
+      <span class='iui-tag-basic' id={id}>
+        {title}
```

Comment:
This `title` prop is definitely the wrong way to go about it. You should be using a `<slot />` instead.

Reply:
sweet i'll use slot instead